content/mdnotes.js

```diff
@@ -303,18 +303,20 @@ function getPDFFileLink(attachment) {
 }
 
 function getZoteroAttachments(item) {
+  const linkStylePref = getPref("pdf_link_style");
   let attachmentIDs = item.getAttachments();
   var linksArray = [];
   for (let id of attachmentIDs) {
     let attachment = Zotero.Items.get(id);
     if (attachment.attachmentContentType == "application/pdf") {
-      let linkContent;
-      if (getPref("pdf_link_style") === "zotero") {
-        linkContent = getZoteroPDFLink(attachment);
+      let link;
+      if (linkStylePref === "zotero") {
+        link = `[${attachment.getField("title")}](${getZoteroPDFLink(attachment)})`;
+      } else if (linkStylePref === "plain") {
```

Comment:
Could you use the [`formatInternalLink`](https://github.com/argenos/zotero-mdnotes/blob/master/content/mdnotes.js#L265) method so that it's consistent with the rest?